scalajvm/app/views/meta/quoteEntity.scala.html

```diff
@@ -0,0 +1,7 @@
+@(quote: models.data.Quote)(implicit requestHeader: play.api.mvc.RequestHeader)
+
+@entity(s"Log List ⚙ Цитата #${quote.id}",
+        "loglist:quote",
+        routes.Assets.at("images/gear_400x400.png").absoluteURL(),
```

Comment:
Nah, too much of an effort. Let's create a separate issue for that and implement it if it's really needed.